### ISSUE TITLE
WIP: Bringing back SELinux

### ISSUE THREAD

**Issue by joelmaxuel** [Sohamlad7/android_device_motorola_cedric#25]
I have been working on this portion and got the device bootable while enforcing again.

I can keep working away on it, just figure now is a good time to have you aware of the effort (obviously you are welcome to help clean it up as well) even though it is not ready to pick at this time.

https://review.carbonrom.org/c/CarbonROM/android_device_motorola_cedric/+/8802

Cheers.

**Comment by Sohamlad7** [comment]
Well even i got it booted with enforcing selinux but WiFi and GPS seems to be broken

**Comment by joelmaxuel** [comment]
Then I guess you are ahead of me.  :grinning:

I cannot speak for GPS as I was not close enough outside, but for me fingerprint reader and camera stopped working too.

I suppose my opening this was a means of introduction and offer of contribution, since I did not see anything new on the SELinux front since the fall when the includes were turned off (and a series of neverallows were added to AOSP around that time).

Although my opening an issue may not be the best way to introduce, if there are better/preferred means to chat.
